lib/error.js

```diff
@@ -131,12 +131,17 @@ function isRetryableError(error) {
  * @class
  * @param {Error|string|object} message The error message
  * @property {string} message The error message
+ * @property {object} result The result document (provided if ok: 1)
  * @return {MongoWriteConcernError} A MongoWriteConcernError instance
  * @extends {MongoError}
  */
-const MongoWriteConcernError = function(message) {
-  MongoError.call(this, message);
+const MongoWriteConcernError = function(message, result) {
+  MongoError.call(this, message, result);
```

Comment:
Why do we need to pass `result` to `MongoError`? It doesn't appear to do anything with it.

Reply:
oops, thanks for catching that!